src/modules/metadata/actions.js

```diff
@@ -62,7 +62,7 @@ export const clearDatasetDataType = networkAction((datasetId, dataTypeID) => (_d
 const fetchProjects = networkAction(() => (_dispatch, getState) => ({
   types: FETCH_PROJECTS,
   url: `${getState().services.metadataService.url}/api/projects`,
-  publicEndpoint: true,
+  publicEndpoint: false,
```

Comment:
this defaults to false-y, so i'd just cut this line out. I'm fine with merging this before the rest, since there's no harm in passing the token to katsu even if it goes unused for now.